tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/javascript_xbrowser_firefox.pysnap

```diff
@@ -71,7 +71,7 @@ app-depth-3:
 --------------------------------------------------------------------------
 app-depth-4:
   hash: "01b5dfbc3a7cf58591f53b2da01832da"
-  tree_label: "aha | callAnotherThing | callback"
+  tree_label: "aha | callAnotherThing | callback | test/<"
```

Comment:
`test/<` represents a frame where grouping discarded the function name and decided to group by filename instead. @jan-auer we can show filename here as fallback? `test.html`

Reply:
I've decided that it's best to hide the frame entirely rather than show a filename that is potentially too long or generic. In the future we may change that behavior again to be closer to package fallback for native but we're not solving for a concrete case we have right now by doing that.

Reply:
sounds good, thanks for clarifying.